package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "ng": "node --max_old_space_size=3000 ./node_modules/.bin/ng",
     "start": "npm run ng serve",
+    "serve:build": "npm run build:libs && npm run build && ./node_modules/http-server/bin/http-server -p 4200 -c-1 dist/make-a-vis-demo",
```

Comment:
This doesn't need to do the building along with the serving. Just serve. Also you should be able to refer to http-server like this: `"serve:build": "http-server -p 4200 -c-1 dist/make-a-vis-demo"`.

Reply:
but to refer http-server, we would have to npm install http-server globally